src/components/FloatingDropdown/FloatingDropdown.st.css

```diff
@@ -0,0 +1,41 @@
+:import {
+    -st-from: "../../common/formatters.st";
+    -st-named: color;
+}
+
+:import {
+    -st-from: "./FloatingDropdownBase.st.css";
+    -st-default: FloatingDropdownBase;
+}
+
+:import {
+    -st-from: "../Dropdown/Dropdown.st.css";
+    -st-named: dropdown;
```

Comment:
@jonathanadler I just re-use all the stylings from local <Dropdown/> component, is that OK?